### DISCUSSION TITLE
Close button placement inconsistency throughout of UI

### DISCUSSION THREAD

**Discussion by Solarunit** [home-assistant/frontend#15698] [Dashboard]
I noticed that close button either on the left or right side throughout of UI. I personally would like to have it only on the right side.

Screenshots:

![](https://i.imgur.com/gqmivL0.png)

![](https://i.imgur.com/Yj8ES2f.png)

**Comment by ildar170975** [comment]
I have already seen a similar issue - ether in “Issues” or in “Discussions”.
Agree, the Close button should be persistent.
